### DISCUSSION TITLE
Pacstall

### DISCUSSION THREAD

**Discussion by trymeouteh** [mjakeman/extension-manager#182] [Ideas]
Please add this app into the pacstall repository. Pacstall is the AUR for Ubuntu which will allow Ubuntu users to easily install the app and receive updates very shortly after an update is released.

https://pacstall.dev/

**Comment by mjakeman** [comment]
Hi, Extension Manager is [already packaged](https://launchpad.net/ubuntu/+source/gnome-shell-extension-manager) in Ubuntu 22.04 and later as part of the `universe` repository (and generously maintained by Canonical themselves). So I personally see little benefit to having a pacstall package.

That being said, I'm happy if someone else would like to package it - some of the [other distribution packages](https://repology.org/project/extension-manager/versions) may be helpful to look at.
